src/dataCorrectionsTool/surveyResetTool.js

```diff
@@ -0,0 +1,285 @@
+                            <div>
+                                <button type="button" class="btn btn-secondary" id="backToToolSelect"><- Back</button>
+                                <button type="button" class="btn btn-danger" id="clearSurveySelect" style="margin-left: 0.5rem;">Clear</button>
+                            </div>
+                            <div style="margin-left: 3rem;">
+                                <button type="button" class="btn btn-primary" id="submitButton" data-toggle="modal" data-target="#modalConfirmReset">Submit</button>
+                            </div>
+                        </div>
+                    </div>
+                </div>    
+            </div>
+        </div>
+
+        <!-- Confirmation Modal -->
+        <div class="modal fade" id="modalConfirmReset" tabindex="-1" aria-labelledby="confirSurveyResetModal" aria-hidden="true">
+            <div class="modal-dialog">
+                <div class="modal-content">
+                <div class="modal-header">
+                    <h5 class="modal-title" id="confirmModalHeader">Confirm Survey Reset</h5>
+                    <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                    </button>
+                </div>
+                <div class="modal-body">
+                    
+                </div>
+                <div class="modal-footer">
+                    <button type="button" class="btn btn-danger" data-dismiss="modal">Cancel</button>
+                    <button type="button" class="btn btn-primary" id="confirmResetButton" data-dismiss="modal">Confirm</button>
+                </div>
+                </div>
+            </div>
+        </div>
+    `;
+};
+
+
+const handleSurveyTypeChange = (participant) => { 
+    const surveyDropdown = document.getElementById('dropdownSurveyMenu');
+    if (!surveyDropdown) return;
+
+    const dropdownSurveyOptions = document.querySelector('#dropdownSurveyMenu').children;
+    if (!dropdownSurveyOptions) return;
+
+    const selectButton = document.querySelector('.selectButton');
+    if (!selectButton) return;
+
+    const participantConnectId = participant['Connect_ID'];
+
+    let query;
+
+    for (let option of dropdownSurveyOptions) {
+        option.addEventListener('click', async (e) => {
+            selectedSurvey = e.target.dataset.survey;
+            if (selectedSurvey === 'ssn') {
+                selectButton.textContent = e.target.textContent;
+                selectedSurvey = e.target.dataset.survey;
+                try {
+                    query = `connectId=${participantConnectId}`
+                    showAnimation();
+                    const response =  await findParticipant(query);
+                    hideAnimation();
+                    const latestParticipant = response.data[0];
+                    localStorage.setItem('particpant', JSON.stringify(latestParticipant));
```

Comment:
I notice this key is `particpant` and not `participant`; is that persisting an existing typo in the key?